src/ServicePulse.Host.Tests/VerifyAppConstantsJSTextReplacement.cs

```diff
@@ -19,9 +21,9 @@ public class VerifyAppConstantsJSTextReplacement
         [Test]
         public void app_constants_js_validation()
         {
-            var pathToConfig = Path.Combine("app.constants.js");
+            var pathToConfig = Path.Combine(TestContext.CurrentContext.TestDirectory, "app.constants.js");
```

Comment:
I added it as after updating to nunit v3 r# test runner changed working directory to: 
C:\Users\Michał\AppData\Local\JetBrains\Installations\ReSharperPlatformVs14_000 which caused this test to fail. Let me verify if it worked with nunit test runner